src/modules/RootPart.rb

```diff
@@ -1663,6 +1676,24 @@ def MountPartitions(root_device_current)
           end
         end
 
+        if has_pam_mount
+          warning = Builtins.sformat(
+            _(
+              "Some home directories in the system on %1 are encrypted. This release does not\n" +
+                "support cryptconfig any longer and those home directories will not be accessible\n" +
+                "after upgade. In order to access these home directories, they need to be decrypted\n" +
+                "before performing upgrade.\n" +
+                "You can consider encrypting whole volume via LUKS."
+            ),
+            root_device_current
+          )
+          if Mode.autoupgrade
```

Comment:
Why not using `Report.Warning`? It will take care of the timeout (if the user configured AutoYaST to do so) and it will work as a Popup when running a regular installation.

Reply:
I made the warning handling the same as for reiserfs